odoo_install.sh

```diff
@@ -260,4 +361,7 @@ echo "Addons folder: $OE_USER/$OE_CONFIG/addons/"
 echo "Start Odoo service: sudo service $OE_CONFIG start"
 echo "Stop Odoo service: sudo service $OE_CONFIG stop"
 echo "Restart Odoo service: sudo service $OE_CONFIG restart"
+if [ $INSTALL_NGINX = "True" ]; then
```

Comment:
Can be moved to upper part (after `sudo service nginx reload`) to skip checking  `$INSTALL_NGINX = "True"`